tests/cylc-cat-log/01-remote.t

```diff
@@ -24,7 +24,10 @@ if [[ -z $CYLC_TEST_HOST ]]; then
     skip_all '"[test battery]remote host": not defined'
 fi
 set_test_number 14
-export CYLC_CONF_PATH=
+create_test_globalrc '' $"
```

Comment:
Single quotes prevent shell variable evaluation in the string (so I should be consistent by using double double-quotes for both, but it looks a bit messier for the empty string case:`""`!)